storage/coop/rchain/key-value-store/KeyLexer.scala

```diff
@@ -95,30 +95,24 @@ class KeyLexer(lineIn: String) {
     var endOfString = false
 
     while (i < line.length && c != '(' && c != ')' && c != ',') {
-      c match {
-        case _ if Character.isLetter(c) => {
-          if (isDigits)
-            throw new Exception(
-              "KeyLexer.NextToken(): malformed (4): '"
-                + line + "', " + i)
-          isLetters = true
-          tokenStr ++= line(i).toString
-        }
-
-        case _ if Character.isDigit(c) => {
-          if (isLetters)
-            throw new Exception(
-              "KeyLexer.NextToken(): malformed (5): '"
-                + line + "', " + i)
-          isDigits = true
-          tokenStr ++= line(i).toString
-        }
-
-        case _ =>
+      if (Character.isLetter(c)) {
+        if (isDigits)
           throw new Exception(
-            "KeyLexer.NextToken(): malformed (6): '"
+            "KeyLexer.NextToken(): malformed (4): '"
               + line + "', " + i)
-      }
+        isLetters = true
+        tokenStr ++= line(i).toString
+      } else if (Character.isDigit(c)) {
+        if (isLetters)
+          throw new Exception(
+            "KeyLexer.NextToken(): malformed (5): '"
+              + line + "', " + i)
+        isDigits = true
+        tokenStr ++= line(i).toString
+      } else
```

Comment:
This line of code makes my head explode. I can't tell in github's diff tool whether or not the the end-brace closes the if clause or the while loop.